Sources/AppcuesKit/AppcuesKit.docc/3.0MigrationGuide.md

```diff
@@ -0,0 +1,11 @@
+# 2.x to 3.0 Migration Guide
+
+## Overview
+
+Updating to this release will not require any code changes for most SDK installations.  Code changes would only be required if your app implemented custom experience traits or actions.
+
+## General Changes
+
+### Removed
+
+- To simplify the public API, `AppcuesExperienceAction` and `AppcuesExperienceTrait` related classes and functions no longer have public visibility. There no current use cases for extensibility that require these to be public.
```

Comment:
> There no current use cases

There **are** no current use cases

I see this is an error from the Android side, I'll fix that!

Reply:
Fixed this